src/game/input.c

```diff
@@ -112,7 +112,6 @@ void Input_Update(void)
     g_Input.menu_down |= g_Input.back;
     g_Input.menu_left |= g_Input.left;
     g_Input.menu_right |= g_Input.right;
-    g_Input.menu_confirm |= g_Input.action;
```

Comment:
This is a breaking change for keyboard players IMO - personally, I use action all the time in menus, so I would have to re-learn using enter. Is there a way to allow KB players to retain action for menus?

Reply:
Ah good catch. I only tested controller.

Reply:
Sure thing, moved the check to S_Input_GetCurrentState so action only doubles as menu_confirm for keyboard players

Reply:
That solution makes sense to me.